File: 009/solve.py
#!usr/bin/python

max = 1000

for a in xrange(1, max/3):
    for b in xrange(a+1, (max-a)/2):
        c = max-b-a
        if a**2+b**2 == c**2:
            print a*b*c

File: 009/solve.go
package main

import "fmt"

func main() {
	max := 1000

	for a := 1; a < max/3; a++ {
		for b := a + 1; b < (max-a)/2; b++ {
			c := max - a - b

			if a*a+b*b == c*c {
				fmt.Println(a * b * c)
				return
			}
		}
	}
}
